src/matchmaking/output/output_fastchess.hpp

```diff
@@ -19,73 +20,127 @@ class Fastchess : public IOutput {
 
     void printElo(const Stats& stats, const std::string& first, const std::string& second,
                   std::size_t current_game_count) override {
-        const Elo elo(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                      stats.penta_LD, stats.penta_LL);
-
-        std::stringstream ss;
-        ss << "Score of "   //
-           << first         //
-           << " vs "        //
-           << second        //
-           << ": "          //
-           << stats.wins    //
-           << "W - "        //
-           << stats.losses  //
-           << "L - "        //
-           << stats.draws   //
-           << "D ["         //
-           << Elo::getScoreRatio(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                                 stats.penta_LD, stats.penta_LL)  //
-           << "] "                                                //
-           << current_game_count                                  //
-           << "\n";
-
-        ss << "Elo difference: "   //
-           << elo.getElo()         //
-           << ", "                 //
-           << "nElo difference: "  //
-           << elo.getnElo()        //
-           << ", "                 //
-           << "LOS: "              //
-           << Elo::getLos(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                          stats.penta_LD, stats.penta_LL)  //
-           << ", "                                         //
-           << "PairDrawRatio: "                            //
-           << Elo::getDrawRatio(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                                stats.penta_LD, stats.penta_LL)  //
-           << "\n";
-
-        std::cout << ss.str() << std::flush;
+        options::Tournament tournament;
```

Comment:
well ofc it doesnt work :D you just create a new tournament config here which has the default values. @gahtan-syarif

Reply:
ahh i see, so it doesnt access the updated values